src/MSALWrapper/MSALWrapper.csproj

```diff
@@ -15,9 +15,6 @@
     <PackageTags>MSAL;authentication</PackageTags>
 
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-
-    <!-- Stylecop warnings as errors flag for build to fail -->
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>

```

Comment:
My concern about removing them as errors is to deviate from stylecop completely and defies the purpose of adding them for code consistency. I get the annoyance though.

Reply:
In this case it's actually required. StyleCop doesn't know how to handle `record` declarations and it explodes. We could disable the single Rule, but I don't think we're at risk of deviating since GitHub does a good job of showing all the warnings inline in the diff. It's much more effective to be able to write code with the warnings, get it working, and then clean it up :)

Reply:
Agreed